opentechinstitute/commotion-docs
# Adding basic responsive design styles

This pull request adds responsive design CSS for the website, targeting three screen sizes:
- above 600px wide
- up to 600px wide
- below 350px wide

The large number of files changed here is an artifact of initially including the _site directory in my commit. Note that it's better practice to not track the _site directory with git. 

TO TEST:
Check out the **issue-33** branch, build & preview locally with **sudo jekyll serve**
Addresses: https://github.com/opentechinstitute/commotion-docs/issues/33
